include/cantera/oneD/StFlow.h

```diff
@@ -78,12 +79,15 @@ class StFlow : public Domain1D
     //! set the transport manager
     void setTransport(Transport& trans);
 
-    //! Enable thermal diffusion, also known as Soret diffusion.
-    //! Requires that multicomponent transport properties be
-    //! enabled to carry out calculations.
-    void enableSoret(bool withSoret) {
-        m_do_soret = withSoret;
-    }
+    //! set the transport manager
+    /*!
+     * @deprecated The withSoret argument is deprecated and unused.
+     *     Use the form of setTransport with signature setTransport(Transport& trans).
+     *     To be removed after Cantera 2.3.
+     */
+    void setTransport(Transport& trans, bool withSoret);
+
+    void enableSoret(bool withSoret);
```

Comment:
It looks like this code is not based on the most current revision of the master branch. Please rebase onto at least 66ba4b5b507fd182a35f0b3310d22fe9ee72e51c and fix any merge conflicts. You're actually removing things that have been specifically added here.